python_modules/libraries/dagster-dg/dagster_dg/cli/__init__.py

```diff
@@ -21,10 +23,12 @@ def create_dg_cli():
     @click.group(
         name="dg",
         commands={
-            "code-location": code_location_group,
-            "component": component_group,
-            "component-type": component_type_group,
-            "deployment": deployment_group,
+            "check": check_group,
+            "docs": docs_group,
+            "info": info_group,
```

Comment:
i will be pedantic and point out that 'info' is not a verb

Reply:
(seems fine though)

Reply:
Should `info` even be here? It was pretty much a placeholder until docs was a thing.

Reply:
I think `introspect` or `inspect` makes more sense, if we should even have it at all. I think it will only be useful for tooling as it is currently implemented. I vote to hide it for now.

Reply:
`inspect` is good. I think we should have it for CLI-focused output, and I'm sure we'll want to inspect many more things. Will update to `inspect` in a follow-up.